use icons for status (#1817)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

--><!-- END: New Entry Template -->

<!-- BEGIN: What's New Entries -->
## August 30 - September 3, 2021

- Published [Notifications page](/components/notification).

> - Published [Toast Notifications](/templates/toast-notifications).
> along with an example.

Reviewer: ```suggestion
- Published [Toast Notifications](/templates/toast-notifications).
```

Reviewer: Let me know when either the "extra period" or dangling sentence is removed. I will approve and merge after that.

> - Published [Notifications page](https://design-system.hpe.design/components/notification) 
> show the first variant of using [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications)

Reviewer: Either link to the "first variant" using "/components/notification#toast-notification" or separate.

```suggestion
- Published [Notifications page](/components/notification).
- Published [Toast Notifications](/templates/toast-notifications).
```

```suggestion
- Published [Notifications page](/components/notification) 
showing the [first variant](/components/notification#toast-notification) using [Toast Notifications](/templates/toast-notifications).
```

> - Publshed [Status Indicators](https://design-system.hpe.design/templates/status-indicator) template page.

Reviewer: ```suggestion
- Published [Status Indicators](https://design-system.hpe.design/templates/status-indicator) template page.
```

> - Updated table example to use the new Status Indicators guidance.

Reviewer: ```suggestion
- Updated all examples referencing "status" to apply the new [Status Indicators guidance](/templates/status-indicator#what-makes-up-a-status-indicator).
```
